editor/doc_tools.cpp

```diff
@@ -346,6 +346,15 @@ void DocTools::generate(bool p_basic_types) {
 
 			prop.overridden = inherited;
 
+			prop.overrides = "";
```

Comment:
It's defined as `String` so it's already an empty string when you create a new struct instance.